buildSrc/updateLibs.js

```diff
@@ -59,3 +65,36 @@ async function roll(src, target) {
 	const bundle = await rollup({ input: path.join(__dirname, src) })
 	await bundle.write({ file: path.join(__dirname, "../libs", target) })
 }
+
+/**
+ * specifically roll up electron-updater and all its dependencies into a single esm file.
+ * it's importing some electron internals directly, so we made a comprehensive list of
+ * exclusions to not roll up.
+ */
+async function rollUpdater(src, target) {
+	const bundle = await rollup({
+		input: path.join(__dirname, src),
+		external: [
+			"assert",
```

Comment:
aren't most of these anyway node builtins that should be anyway not imported?

Reply:
rollup didn't think so apparently